packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/utils.spec.ts

```diff
@@ -21,59 +25,83 @@ const tableColumnConfig: WfoTableColumnConfig<TestObject> = {
+                    sortableFieldNames,
+                    filterableFieldNames,
+                );
 
-        // Then
-        if (
-            result.name?.columnType === ColumnType.DATA &&
-            result.age?.columnType === ColumnType.DATA
-        ) {
-            expect(result.name.isSortable).toEqual(true);
-            expect(result.name.isFilterable).toEqual(true);
-            expect(result.age.isSortable).toEqual(true);
-            expect(result.age.isFilterable).toEqual(true);
-        } else {
-            // Preventing silently skipping above expects
-            throw Error('Some of the fields are not data fields');
-        }
-    });
-    it('sets the sortable and filterable properties for the columnConfig object to false when the colum name is not specified in the list', () => {
-        // Given
-        const sortableFieldNames = ['nonExistingFieldName'];
-        const filterableFieldNames: string[] = [];
+            // Then
+            if (
+                result.name?.columnType === ColumnType.DATA &&
+                result.age?.columnType === ColumnType.DATA
+            ) {
+                expect(result.name.isSortable).toEqual(true);
+                expect(result.name.isFilterable).toEqual(true);
+                expect(result.age.isSortable).toEqual(true);
+                expect(result.age.isFilterable).toEqual(true);
+            } else {
+                // Preventing silently skipping above expects
+                throw Error('Some of the fields are not data fields');
+            }
+        });
+        it('sets the sortable and filterable properties for the columnConfig object to false when the colum name is not specified in the list', () => {
+            // Given
+            const sortableFieldNames = ['nonExistingFieldName'];
+            const filterableFieldNames: string[] = [];
 
-        // When
-        const result =
-            mapSortableAndFilterableValuesToTableColumnConfig<TestObject>(
-                tableColumnConfig,
-                sortableFieldNames,
-                filterableFieldNames,
-            );
+            // When
+            const result =
+                mapSortableAndFilterableValuesToTableColumnConfig<TestObject>(
+                    tableColumnConfig,
+                    sortableFieldNames,
+                    filterableFieldNames,
+                );
 
-        // Then
-        if (
-            result.name?.columnType === ColumnType.DATA &&
-            result.age?.columnType === ColumnType.DATA
-        ) {
-            expect(result.name.isSortable).toEqual(false);
-            expect(result.name.isFilterable).toEqual(false);
-            expect(result.age.isSortable).toEqual(false);
-            expect(result.age.isFilterable).toEqual(false);
-        } else {
-            // Preventing silently skipping above expects
-            throw Error('Some of the fields are not data fields');
-        }
+            // Then
+            if (
```

Comment:
TypeScript is unable to determine the column type of the name and age columns when it comes back from the mapper function, the check is needed to avoid TSC errors. We need the else branch to make the unit test fail when skipped.